chatkit-core/src/integrationTest/kotlin/com/pusher/chatkit/TestChatkitDependencies.kt

```diff
@@ -24,31 +28,20 @@ class TestDependencies : PlatformDependencies {
         override fun warn(message: String, error: Error?) = log("W", message.take(MAX_LOG_LENGTH), error)
         override fun error(message: String, error: Error?) = log("E", message.take(MAX_LOG_LENGTH), error)
         private fun log(type: String, message: String, error: Error?) =
-                println("${ts()} $type: $message ${error?.let { "\n" + it }
-                        ?: ""}".take(MAX_LOG_LENGTH))
+            println("${ts()} $type: $message ${error?.let { "\n" + it }
+                ?: ""}".take(MAX_LOG_LENGTH))
 
         private fun ts() = timeFormatter.format(LocalDateTime.now())
     }
     override val mediaTypeResolver: MediaTypeResolver = object : MediaTypeResolver {
         override fun fileMediaType(file: File): String? = "image/jif"
     }
     override val sdkInfo: SdkInfo = SdkInfo(
-            product = "SynchronousChatManager Integration Tests",
-            language = "Spek",
-            platform = "JUnit",
-            sdkVersion = "test"
+        product = "Chatkit Tests",
+        sdkVersion = SDK_VERSION,
+        platform = "JVM",
+        language = "Kotlin"
     )
 }
 
-class TestChatkitDependencies(
-    override val tokenProvider: TokenProvider,
-    platformDependencies: PlatformDependencies = TestDependencies()
-) : ChatkitDependencies, PlatformDependencies by platformDependencies {
-    override val okHttpClient: OkHttpClient = insecureOkHttpClient.newBuilder().apply {
-        addInterceptor { chain ->
-            chain.proceed(chain.request().newBuilder().addHeader("Connection", "close").build())
-        }
-    }.build()
-    override val pushNotifications = NoPushNotificationFactory()
-    override val appHooks = NoAppHooks()
-}
+private const val MAX_LOG_LENGTH = 3000
```

Comment:
does it make sense for this val to be moved to the bottom of the file? don't we want consts at the top?

Reply:
it's at the bottom as it's private (impl detail, less important than the rest)